py/tests/external-deps/custom-macro/macro.bzl

```diff
@@ -1,17 +1,12 @@
 # buildifier: disable=module-docstring
-load("//py:defs.bzl", "py_binary", "py_library")
+load("//py:defs.bzl", "py_binary")
 
 def click_cli_binary(name, deps = [], **kwargs):
-    py_library(
-        name = name + "_lib",
-        srcs = ["//py/tests/external-deps/custom-macro:__main__.py"],
-    )
-
     py_binary(
         name = name,
         main = "//py/tests/external-deps/custom-macro:__main__.py",
+        srcs = ["//py/tests/external-deps/custom-macro:__main__.py"],
```

Comment:
note: this is required because this file was previously relying on locating a main among deps rather than srcs. This would not have worked with rules_python, so there shouldn't be user code like this.